src/components/avatar/Avatar.scss

```diff
@@ -38,6 +37,14 @@
         line-height: 1;
     }
 
+    @for $i from 1 through length($avatar-colors) {
+        $backgroundColor: nth($avatar-colors, $i);
```

Comment:
This probably doesn't need to be a variable.

Reply:
Removed